packages/devtools_app/lib/src/screens/network/har_network_data.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:flutter/foundation.dart';
+
+import '../../../devtools_app.dart';
```

Comment:
don't import this file directly, but rather import the implementation files directly

Reply:
done